framework/include/partitioner/PetscExternalPartitioner.h

```diff
@@ -34,10 +34,21 @@ class PetscExternalPartitioner : public MoosePartitioner
 
   virtual dof_id_type computeSideWeight(Elem & elem, unsigned int side);
 
+  using Partitioner::partition;
```

Comment:
Are you sure you want to import the namespace?

Reply:
I added this line because the complier is complaining. I will take any other suggestions if possible.

```
In file included from /opt/civet/build_0/moose/framework/build/header_symlinks/MoosePartitioner.h:17:0,
                 from /opt/civet/build_0/moose/framework/build/header_symlinks/GridPartitioner.h:15,
                 from /opt/civet/build_0/moose/framework/src/partitioner/GridPartitioner.C:10,
                 from /opt/civet/build_0/moose/framework/build/unity_src/partitioner_Unity.C:2:
/opt/civet/build_0/moose/scripts/../libmesh/installed/include/libmesh/partitioner.h:110:16: error: 'virtual void libMesh::Partitioner::partition(libMesh::MeshBase&)' was hidden [-Werror=overloaded-virtual]
   virtual void partition (MeshBase & mesh);
                ^~~~~~~~~
In file included from /opt/civet/build_0/moose/framework/src/partitioner/PetscExternalPartitioner.C:10:0,
                 from /opt/civet/build_0/moose/framework/build/unity_src/partitioner_Unity.C:5:
/opt/civet/build_0/moose/framework/build/header_symlinks/PetscExternalPartitioner.h:37:16: error:   by 'virtual void PetscExternalPartitioner::partition(libMesh::MeshBase&, unsigned int)' [-Werror=overloaded-virtual]
   virtual void partition(MeshBase & mesh, const unsigned int n) override;
                ^~~~~~~~~
cc1plus: all warnings being treated as errors
/opt/civet/build_0/moose/framework/build.mk:97: recipe for target '/opt/civet/build_0/moose/framework/build/unity_src/partitioner_Unity.x86_64-pc-linux-gnu.dbg.lo' failed
make: *** [/opt/civet/build_0/moose/framework/build/unity_src/partitioner_Unity.x86_64-pc-linux-gnu.dbg.lo] Error 1
make: *** Waiting for unfinished jobs....
ERROR: exiting with code 2

```